composer.json

```diff
@@ -25,11 +25,12 @@
         "sonata-project/doctrine-extensions": "^1.0",
         "sonata-project/easy-extends-bundle": "^2.2",
         "swiftmailer/swiftmailer": "^4.3 || ^5.0 || ^6.0",
-        "symfony/console": "^2.8 || ^3.2",
-        "symfony/form": "^2.8 || ^3.2",
-        "symfony/http-foundation": "^2.8 || ^3.2",
-        "symfony/security": "^2.8 || ^3.2",
-        "symfony/translation": "^2.8 || ^3.2"
+        "symfony/console": "^2.8 || ^3.2 | ^4.0",
+        "symfony/form": "^2.8 || ^3.2 | ^4.0",
+        "symfony/http-foundation": "^2.8 || ^3.2 | ^4.0",
+        "symfony/security-core": "^2.8 || ^3.2 | ^4.0",
+        "symfony/security-acl": "^2.8 || ^3.2 | ^4.0",
+        "symfony/translation": "^2.8 || ^3.2 | ^4.0"
```

Comment:
You changed way more things that you said you changed.

Reply:
@greg0ire yes, sorry, add one more line to changes